app/views/posts/_form.html.erb

```diff
@@ -17,12 +17,17 @@
   </div>
   
   <div class="field">
-    <%= f.file_field :picture %> 
+    <%= f.label :body %><br>
+    <%= f.text_area :body %>
   </div>
   
   <div class="field">
-    <%= f.label :body %><br>
-    <%= f.text_area :body %>
+    <%= f.label :tags %><br>
+    <%= f.text_area @post.tags %>
```

Comment:
to jest zle, normalnie bys uzyl `f.text_area :post, :pole_w_post`, ale `tags` nie jest polem w `Post`, wiec nie mozesz tak zrobic.
Najprosciej uzyc tutaj `text_area_tag` (bez `f.` na poczatku). Obczaj sobie doc: http://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-text_area_tag

Twoim celem jest wyslanie w parametrach wartosci text_area jako klucz `tags`.